app/root.tsx

```diff
@@ -137,6 +138,7 @@ export default function App() {
 		</html>
 	)
 }
+export default withSentry(App);
```

Comment:
What happens here if people never set up sentry?

Reply:
@kentcdodds if you don't have a DSN set, Sentry basically becomes a no-op. Everything will work as usual.